src/prompts/grade.ts

```diff
@@ -166,6 +167,28 @@ function createInput(data: Buffer): CoreMessage {
   };
 }
 
+if (!process.env.ENCRYPTION_KEY) {
+  throw new Error("ENCRYPTION_KEY must be set in the env file");
```

Comment:
Might be better to skip the encrypted coding if the encryption key is missing or incorrect, so errors don't take down productrion.